src/gsudo/Native/ProcessApi.cs

```diff
@@ -227,7 +227,7 @@ internal static extern bool CheckRemoteDebuggerPresent(
 
 
         [DllImport("kernel32.dll", CharSet = CharSet.Auto, SetLastError = true)]
-        internal static extern bool CreatePipe(out SafeFileHandle hReadPipe, out SafeFileHandle hWritePipe, SECURITY_ATTRIBUTES lpPipeAttributes, int nSize);
+        internal static extern bool CreatePipe(out SafeFileHandle hReadPipe, out SafeFileHandle hWritePipe, IntPtr lpPipeAttributes, uint nSize);
```

Comment:
You can also use “ref SECURITY_ATTRIBUTES” if you ever use it.
I recommend Vanara library if you see there are many WinAPIs you have to define. 
https://github.com/dahall/Vanara